bitnami/kube-prometheus/README.md

```diff
@@ -516,6 +515,13 @@ Find more information about how to deal with common errors related to Bitnami’
 $ helm upgrade my-release bitnami/kube-prometheus
 ```
 
+### To 4.4.0
+
+This version replaced the old `configReloaderCpu` and `configReloaderMemory`
+variables in favor of the new `configReloaderResources` map to define the
+requests and limits for the config-reloader sidecards. Users who made use
+of the old variables will need to migrate to the new ones.
```

Comment:
Thanks for including this note! You don't need to split the text into 4 lines since GitHub automatically adjust the text by the width.
```suggestion
This version replaced the old `configReloaderCpu` and `configReloaderMemory` variables in favor of the new `configReloaderResources` map to define the requests and limits for the config-reloader sidecards. Users who made use of the old variables will need to migrate to the new ones.
```